### ISSUE TITLE
question about result

### ISSUE THREAD

**Issue by an1018** [JiahuiYu/generative_inpainting#173]
I've run one epoch on places2 dataset. The loss and result of  repair are as follows. Do you think I should run more epoches?
![image](https://user-images.githubusercontent.com/19808900/48321914-dc790380-e65f-11e8-9978-c34d76e674bd.png)
![image](https://user-images.githubusercontent.com/19808900/48321995-3ed20400-e660-11e8-8881-88b91941528f.png)

**Comment by an1018** [comment]
@JiahuiYu Do I need to modify hyper-parameters,and retrain a new model?

**Comment by JiahuiYu** [comment]
@an1018 You can just resume training.

**Comment by an1018** [comment]
@JiahuiYu I've just started learning GAN, can you tell me which parameters I need to modify and how much it should be changed?Thank you very much. I look forward to your reply.
